wave_matching/src/icp.cpp

```diff
@@ -13,17 +13,20 @@ ICPMatcher::ICPMatcher(float res) {
     } else {
         this->resolution = -1;
     }
-    wave::ConfigParser parser;
+    ConfigParser parser;
```

Comment:
that's fair, leave it for now and if we need to add more constructors later, we can do that.